### ISSUE TITLE
publish latest beta releases?

### ISSUE THREAD

**Issue by tomcl** [fable-compiler/Fable#2807]
### Description

Please provide a succinct description of your issue.

It is not easy to know when a new release exists because it is not published on this repo (release stuck at 3.7.0-beta-003 when latest release is 3.7.0-beta-014).

It matters because bugs in watch can be a pain for everyone - they get fixed in later versions.

**Comment by alfonsogarciacaro** [comment]
Yes, sorry for that! Not the best practice but I was "testing" the betas in a private project so I didn't make them very public 😅 

I just pushed the _officialish_ 3.7 release. Can you please try and let us know if you find any issue? Thank you!
